--- ui.c	2001-04-26 00:04:37.000000000 +0800
+++ ui1.c	2010-01-07 19:24:58.265860858 +0800
@@ -18,11 +18,11 @@
 }
 static void do_display(void)
 {
-    aa_puts(context, 0, 0, AA_SPECIAL, "Calculating..");
-    aa_flush(context);
+    //aa_puts(context, 0, 0, AA_SPECIAL, "Calculating..");
+    //aa_flush(context);
     shrink(context, fast, x1, y1, x2, y2);
     aa_render(context, renderparams, 0, 0, aa_scrwidth(context), aa_scrheight(context));
-    aa_flush(context);
+    //aa_flush(context);
 }
 static void resize(aa_context * c)
 {
@@ -204,6 +204,8 @@
 	    resized = 0;
 	}
 	aa_flush(context);
+    quit = 1;
+    break;
 	c = aa_getevent(context, 1);
 	switch (c) {
 	case 'h':
